native/tools/native-locales

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const { program } = require('commander')
+
+const setupNativeLocales = (appName) => {
+  const { native: nativeLocales } = JSON.parse(fs.readFileSync('../../locales/locales.json'))
+  const languageCodes = Object.keys(nativeLocales)
+  console.warn('Creating InfoPlist.strings for the languages ', languageCodes)
+
+  languageCodes.forEach(language => {
+    const locales = nativeLocales[language]
+    const keys = Object.keys(locales)
+
+    const content = keys.map((key) => {
+      const regex = /{\{appName}}/gi
```

Comment:
Why are here double closing brackets? `}}`

Reply:
Because it should replace `{{appName}}`. The escaping is probably wrong though.

Reply:
The escape was redundant, removed.